VncSharp/RfbProtocol.cs

```diff
@@ -19,6 +19,7 @@
 using System.Diagnostics;
 using System.Drawing;
 using System.IO;
+using System.Linq;
```

Comment:
What's this for? (mostly curious, fine if you need it.  I'm not clear why it's being added)

Reply:
This is used for the following (Reverse):
BitConverter.GetBytes((ushort)x).Reverse().ToArray().CopyTo(buff, 2);

Reply:
Thanks.  .NET has evolved a lot since I wrote this code, so I wondered.

Reply:
No problem :-)